src/databricks/labs/lakebridge/resources/assessments/synapse/pipeline_config.yml

```diff
@@ -0,0 +1,69 @@
+name: synapse_assessment
+version: "1.0"
+extract_folder: "/tmp/data/synapse_assessment"
+steps:
+  - name: workspace_info
+    type: python
+    extract_source: src/databricks/labs/remorph/resources/assessments/synapse/workspace_extract.py
+    mode: overwrite
+    frequency: once
+    flag: active
+    dependencies:
+      - pandas
+      - duckdb
+      - azure-synapse-artifacts
+      - azure-identity
+      - azure-monitor-query==1.3.0b1
+      - databricks-sdk
+      - databricks-labs-blueprint[yaml]>=0.10.1
+      - sqlalchemy
+      - pyodbc
+  - name: monitoring_metrics
+    type: python
+    extract_source: src/databricks/labs/remorph/resources/assessments/synapse/monitoring_metrics_extract.py
```

Comment:
Same here and remaining extract source attributes - the path is off after the directory rename from `remorph` -> `lakebridge`